packages/chain-adapters/src/solana/SolanaChainAdapter.ts

```diff
@@ -466,6 +467,7 @@ export class ChainAdapter implements IChainAdapter<KnownChainIds.SolanaMainnet>
       chainSpecific.addressLookupTableAccounts ?? [],
     )
 
+    if (isUndefined(input.to)) throw new Error(`${this.getName()}ChainAdapter: value is required`)
```

Comment:
```suggestion
    if (isUndefined(input.to)) throw new Error(`${this.getName()}ChainAdapter: to is required`)
```

Reply:
Fixed in [0001e53](https://github.com/shapeshift/web/pull/8120/commits/0001e5316ccd4395724ab75e36a101390df9b1d8)